cli_tools/gce_ovf_import/ovf_importer/ovf_importer.go

```diff
@@ -363,8 +363,13 @@ func (oi *OVFImporter) modifyWorkflowPostValidate(w *daisy.Workflow) {
 	rl.LabelResources(w)
 	daisyutils.UpdateAllInstanceNoExternalIP(w, oi.params.NoExternalIP)
 	if oi.params.UefiCompatible {
```

Comment:
Since --uefi-compatible is not a gcloud flag, we can deprecate it in wrappers if auto UEFI tagging works